src/components/ha-selector/ha-selector-number.ts

```diff
@@ -36,27 +36,23 @@ export class HaNumberSelector extends LitElement {
           >
           </ha-slider>`
         : ""}
-      <paper-input
+      <mwc-textfield
```

Comment:
Not new in this PR, but can we do something about the width of slider vs input?

![image](https://user-images.githubusercontent.com/1444314/152620983-efdb910a-c17e-4b04-8785-90b66fdf048e.png)

Reply:
Not seeing that? 
![image](https://user-images.githubusercontent.com/5662298/152648299-ca4be427-938e-4f0e-8360-5f84da822531.png)

Firefox only?

Reply:
Yes... Firefox only